### ISSUE TITLE
cloudos is greedy

### ISSUE THREAD

**Issue by keiranmraine** [VertebrateResequencing/wr#200]
In my .wr_config.yml I have set:

```
cloudos: "cgp-ds_ubuntu_2019-04-01_2.2.0"
```

When the manager starts it uses: `cgp-ds_ubuntu_2019-04-01_2.2.0-1-ga5f7897-dirty` even though an exact match is present.

I'm aware that the docs indicate this is a prefix, but should it take an exact match in preference?  I see the current behaviour being a potential pain point.

**Comment by sb10** [comment]
I agree that an exact match is preferable. In the mean time, you can use the image id instead of name to ensure the correct image is chosen.

**Comment by keiranmraine** [comment]
Could you update the docs to indicate that image id works too, currently it reads like only name is possible:

https://github.com/VertebrateResequencing/wr/blob/70758745c50add648c3a255e51a066bfcc17ae6e/wr_config.yml#L404-L411

Something like:

```
 # cloudos: What OS image should be used for spawned servers? 
 # This defaults to "Ubuntu Xenial". It is overridden by the --os option to 
 # `wr cloud deploy` and the --cloud_os option of `wr manager start`. 
 # Note, this is the string prefix of an image name that is available to you
 # OR an image id
```

**Comment by sb10** [comment]
Implemented in PR #202
